components/service/glean/src/test/java/mozilla/components/service/glean/storages/TimingDistributionsStorageEngineTest.kt

```diff
@@ -226,7 +226,10 @@ class TimingDistributionsStorageEngineTest {
 
     @Test
     fun `getBuckets() correctly populates the buckets property`() {
-        // Hand calculated values using current default range 0 - 60000 and bucket count of 100
+        // Hand calculated values using current default range 0 - 60000 and bucket count of 100.
+        // NOTE: The final bucket here of 60001 is a product of the bucketing algorithm and
```

Comment:
nit: I would focus more on the purpose of the bucket rather than how it originated. I would (please push back or phrase it better!) say that regardless of the bucket size/width, the last bucket holds samples beyond the max range.

Reply:
:+1: